tests/sagemaker/README.md

```diff
@@ -0,0 +1,153 @@
+# Testing new Hugging Face Deep Learning Container.
+
+This document explains the testing strategy for releasing the new Hugging Face Deep Learning Container. AWS maintains 14 days of currency with framework releases. Besides framework releases, AWS release train is bi-weekly on Monday. Code cutoff date for any changes is the Wednesday before release-Monday. 
+
+
+## Test Case 1: Releasing a New Version of 🤗 Transformers
+
+### Requirements: Test should run on Release Candidate for new `transformers` release to validate the new release is compatible with the DLCs. To run these tests you need credentials for the HF SageMaker AWS Account. You can ask @philschmid or @n1t0 to get access.
```

Comment:
@sgugger tells me this doesn't apply to patches, only to minor/major versions. This could maybe be explicited in the doc?

Reply:
We agreed only on minor/major version. I also suggest that after we released for example `transformers 4.5.0` wait 1-2 days to see how the version behaves and if there is a need for a patch to build the container with the patch included.